keps/sig-node/5526-pod-level-resource-managers/README.md

```diff
@@ -0,0 +1,1767 @@
+that a rollback is necessary:
+
+-   **`pod_topology_manager_pinning_errors_total`**: A rise in this metric
+    indicates that the Topology Manager is failing to find suitable NUMA
+    alignments for pods utilizing pod-level resources. This suggests a
+    fundamental issue with the new hint generation or merging logic.
+-   **`pod_cpu_manager_pinning_errors_total`**: An increase here points to
+    failures in the CPU Manager's new partitioning or allocation logic for
+    pod-level resources. This could manifest as pods failing to acquire their
+    requested CPU sets.
+-   **`pod_memory_manager_pinning_errors_total`**: Similarly, a spike in this
+    metric signals problems with the Memory Manager's new partitioning or
+    allocation for pod-level memory, potentially leading to memory allocation
+    failures.
+-   **`topology_manager_admission_errors_total`**: While an existing metric, a
+    significant increase after enabling the feature could indicate that the
+    Topology Manager is rejecting pods due to issues introduced by the new
+    pod-level resource management, even if the specific
+    `pod_topology_manager_pinning_errors_total` is not yet high.
+-   **`cpu_manager_pinning_errors_total`**: An increase in this existing metric
+    could indicate broader CPU pinning issues, potentially exacerbated or
+    triggered by the new pod-level logic.
+-   **`memory_manager_pinning_errors_total`**: An increase in this existing
+    metric could indicate broader memory pinning issues, potentially exacerbated
+    or triggered by the new pod-level logic.
+
+Additionally, general system health metrics such as node CPU/memory utilization,
+pod startup latency, and overall application error rates should be monitored.
+Any unexpected degradation in these, correlated with the feature enablement,
+would also warrant investigation and potential rollback.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Manual testing of the upgrade/downgrade path will be performed as part of the
+alpha development cycle.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
+
+N/A.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+An operator can determine if the feature is in use by monitoring the new Kubelet
+metrics exposed when the feature is enabled. A non-zero value for any of the
+following metrics is a definitive signal that workloads are actively using the
+pod-level resource management capabilities on a given node:
+
+-   `pod_resource_manager_exclusive_allocation_pods`
```

Comment:
this is probably worth keeping but let's elaborate a bit the signal it conveys

Reply:
I made a new proposal, please refer to https://github.com/kubernetes/enhancements/pull/5527#discussion_r2412389247.